bin/public/index.html

```diff
@@ -1,17 +1,111 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
```

Comment:
Yes, indeed. Should i create a different folder inside public? perhaps 

Themes
-> Layout 1
-> Layout 2

I've one more layout, that im still working on.